Created workflows-ga (#14250)
PostHog/posthog.com: contents/blog/workflows-ga.md

---

> title: Workflows is now GA. Congrats, your product data just got a job

Reviewer: I get the "generally available" is a really long phrase, but I personally don't like "GA" as an acronym, always takes me a second to think of what it stands for and makes me think "GOOGLE ANALYTICS" as my first thought

Not a blocker, but something to think about (or ignore 😅) for the future

featuredImageType: full
category: Product
tags:
    - Workflows
seo:
    {
        metaTitle: 'Workflows is now GA. Congrats, your product data just got a job',
        metaDescription: 'Workflows is now generally available. Trigger messages and actions directly from PostHog events and user data: no syncs, no glue tools, no duct tape.',
    }
---

We’ve always been good at collecting product data. In PostHog, you could see when a user hits an important milestone in your product, but couldn’t act on it. If PostHog already knows what happened, why does the next step have to happen in another system? That question is what led to Workflows.

We introduced [Workflows in alpha](blog/workflows-alpha) asking: could we let teams define actions directly on top of the same events and user data they already trust for analysis? [During the beta](blog/workflows-beta), we saw it used to automate onboarding, coordinate feature rollouts, and trigger messages based on real product behavior. 

With that feedback and usage in place, **[Workflows](https://app.posthog.com/workflows)** is now generally available. 

## What does Workflows do? 

> **[Workflows](https://app.posthog.com/workflows)** lets you automate product-led actions directly from your PostHog data. When something happens in your product, you can respond immediately by sending an email, updating a user property, or triggering a Slack alert, without syncing tools, duct-taped integrations, or API key scavenger hunts.

Reviewer: ```suggestion
**[Workflows](https://app.posthog.com/workflows)** lets you automate product-led actions directly from your PostHog data. When something happens in your product, you can respond immediately by sending an email, updating a person property, or triggering a Slack alert. This is done without multi-product syncs, duct-taped integrations, or API key scavenger hunts.
```

- Assign users to a specific variant
- Update user properties for future targeting

This keeps what users see in the product and what they’re told about it in sync, without relying on separate systems to catch up later.

## Less theory, more shipping

At [Croissant](customers/croissant), Workflows is used to act on intent signals without moving data between tools. Their growth team triggers follow-ups directly from PostHog events, keeping experimentation and iteration fast.

> “The main benefit is that everything’s already in PostHog. Syncing data across tools is always hit-or-miss, and expensive. Now it’s all in one place, and we can iterate way faster.”

At [Grantable](customers/grantable), Workflows builds on an analytics foundation where every feature is instrumented and measured. Product usage data now feeds automation directly, instead of stopping at dashboards.

> “We track usage, pipe that into dashboards for activation and retention, and now we’re starting to act on it automatically with Workflows.”

Different products, same pattern: using trusted product events as the source of truth and acting on them without adding another automation layer to maintain.

> ## Start building with Workflows

Reviewer: This would be a good place to talk about pricing (or is this now ready yet)? How much free tier does someone get? How does it compare to other workflow products?
